### ISSUE TITLE
applyign a task templates kills root

### ISSUE THREAD

**Issue by liorkesos** [linnovate/root#699]
When applying a task template the applicaiton dies and rendered unusable.
This is the error from the docker logs...
```

current user
{"__v":0,"uid":"cad23be0-3671-11e8-af14-33b030ab2801","id":"lior@linnovate.net","name":"lior","email":"lior@linnovate.net","username":"liorkesos","_id":"5ac222abcaa4921100f6add5","GetMailEveryDayAboutMyTasks":"no","GetMailEveryWeekAboutGivenTasks":"no","GetMailEveryWeekAboutMyTasks":"no","provider":"local","roles":["authenticated"]}
/usr/src/app/node_modules/mpromise/lib/promise.js:108
  if (this.ended && !this.hasRejectListeners()) throw reason;
                                                ^

TypeError: Cannot read property '_id' of undefined
    at EventEmitter.<anonymous> (/usr/src/app/packages/custom/icu/server/controllers/templates.js:241:34)
    at EventEmitter.<anonymous> (/usr/src/app/node_modules/mpromise/lib/promise.js:174:48)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)
    at Promise.safeEmit (/usr/src/app/node_modules/mpromise/lib/promise.js:81:21)
    at Promise.reject (/usr/src/app/node_modules/mpromise/lib/promise.js:109:15)
    at Promise.error (/usr/src/app/node_modules/mongoose/lib/promise.js:94:15)
    at Promise.resolve (/usr/src/app/node_modules/mongoose/lib/promise.js:112:24)
    at /usr/src/app/node_modules/mongoose/lib/document.js:1559:39
    at proto.(anonymous function).handleError (/usr/src/app/node_modules/hooks-fixed/hooks.js:40:22)
    at proto.(anonymous function).next_ (/usr/src/app/node_modules/hooks-fixed/hooks.js:75:26)
    at EventEmitter.fnWrapper (/usr/src/app/node_modules/hooks-fixed/hooks.js:171:15)
    at EventEmitter.<anonymous> (/usr/src/app/node_modules/mpromise/lib/promise.js:174:48)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)
    at Promise.safeEmit (/usr/src/app/node_modules/mpromise/lib/promise.js:81:21)
    at Promise.reject (/usr/src/app/node_modules/mpromise/lib/promise.js:109:15)
    at /usr/src/app/node_modules/mongoose/lib/model.js:263:20
    at newTickHandler (/usr/src/app/node_modules/mpromise/lib/promise.js:229:18)
    at nextTickCallbackWith0Args (node.js:436:9)
    at process._tickCallback (node.js:365:13)```

**Comment by pashist** [comment]
Could you provide more details how to reproduce this issue ?

**Comment by liorkesos** [comment]
https://www.useloom.com/share/08729bfa439f47deb12a5b44e8ba0f9f

**Comment by nevos** [comment]
resolved in 00d9bcc038d58029aaa59482207b9980cd5c2f12
